src/app/operators/dialogs/expression-operator/expression-operator.component.html

```diff
@@ -1,54 +1,78 @@
+<style type="text/css">
+    .custom-unit {
+    }
+</style>
 <wave-sidenav-header>Calculate Expression on Raster</wave-sidenav-header>
-<form [formGroup]="form" (ngSubmit)="$event.preventDefault();add($event)" fxLayout="column">
+<form [formGroup]="form" (ngSubmit)="add()" fxLayout="column">
     <div fxFlex class="container">
-        <wave-multi-layer-selection
-            [types]="[ResultTypes.RASTER]"
-            formControlName="rasterLayers"
-            [min]="1" [max]="5"
-        ></wave-multi-layer-selection>
-        <wave-dialog-section-heading
-            title="Configuration" subtitle="Specify the operator"
-        ></wave-dialog-section-heading>
+        <wave-dialog-help>
+            <p>The <em>Expression Operator</em> allows applying mathematical calculations on one or more raster layers.</p>
+            <p>For the expression, each raster is referred to an alphabetic character, e.g., A for the first raster.
+                It is possible to use basic operations like <code>+</code>, <code>-</code>, <code>*</code> and <code>/</code>, e.g., <code>A
+                    + B</code>.</p>
+            <p>It is necessary to specify the output datatype for the expression.
+                The list indicates the datatype of the input.
+                Furthermore, it is required to specify the range of the data in the form of min and max values.
+                The system will choice a appropriate NO DATA value automatically.</p>
+            <p>There is the option to specify an output unit for the expression result, e.g. <em>temperature</em> in <em>°C</em>.
+                This will be shown in the legend and automatically used in plots, etc.</p>
+            <p>Since the input rasters can be in differing projections, it is required to specify the output projection.</p>
```

Comment:
Indicate that this is the projection used when applying the operator.

Reply:
73fdbfd